invenio_search/ext.py

```diff
@@ -81,6 +81,9 @@ def register_mappings(self, alias, package_name):
         # placed at the root level of the specified package path, and not in
         # the `<package-path>/v2` directory.
         if ES_VERSION[0] == 2:
+            warnings.warn(
```

Comment:
I woudl move it down to the ``init_app`` method to ensure that it's run no matter what